support auto rtl detection in browsers (#4464)
mattermost/docs: source/help/messaging/sending-messages.rst

.. note::
  Message edits do not trigger new @mention notifications, desktop notifications, or notification sounds.

Your System Admin `can disable the Edit feature or set a time limit <https://docs.mattermost.com/administration/config-settings.html?highlight=config%20settings#allow-users-to-edit-their-messages>`__ before messages become uneditable, available in Mattermost Enterprise Edition E10 and E20.

Deleting Messages
------------------

> Click the **[...]** link next to a message that you have sent, then click **Delete**.

Reviewer: ```suggestion
Select  the **[...]** link next to a message that you've sent, then select **Delete**.
```
